iidm/iidm-impl/src/test/java/com/powsybl/iidm/network/impl/tck/AliasesTest.java

```diff
@@ -0,0 +1,15 @@
+/*
+ * Copyright (c) 2020, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.iidm.network.impl.tck;
+
+import com.powsybl.iidm.network.tck.AbstractAliasesTest;
+
+/**
+ * @author Sebastien Murgey <sebastien.murgey at rte-france.com>
```

Comment:
In my memories, you looked different :)

Reply:
I only moved this file, I didn't create it, this is why I didn't change the author!